velox/docs/monitoring/stats.rst

```diff
@@ -69,3 +69,27 @@ These stats are reported only by HashBuild and HashAggregation operators.
      - nanos
      - Time spent on building the hash table from rows collected by all the
        hash build operators. This stat is only reported by the HashBuild operator.
+
+Spilling
+--------
+These stats are reported operators that supporting spilling.
```

Comment:
"These stats are reported by operators that support spilling."

BTW shall we include other stats as well? It seems there are only read metrics

Reply:
Fixed.

>BTW shall we include other stats as well? It seems there are only read metrics

Yes, I will document the write stats, and add their keys in the `Operator` class too in the follow-up. What do you think?